<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Google API Shopping Client</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>754</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuProducts">
    <property name="title">
     <string>Products</string>
    </property>
    <addaction name="actionLoad_list"/>
    <addaction name="actionShow_details"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="actionNext"/>
    <addaction name="actionBack"/>
    <addaction name="actionHistogram"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuProducts"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
   <addaction name="actionBack"/>
   <addaction name="actionNext"/>
   <addaction name="actionHistogram"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/exit.png</normaloff>:/images/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionLoad_list">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/searchfind.png</normaloff>:/images/images/searchfind.png</iconset>
   </property>
   <property name="text">
    <string>Load list</string>
   </property>
  </action>
  <action name="actionShow_details">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/details.png</normaloff>:/images/images/details.png</iconset>
   </property>
   <property name="text">
    <string>Show details</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/refresh.png</normaloff>:/images/images/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/next.png</normaloff>:/images/images/next.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/prev.png</normaloff>:/images/images/prev.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionHistogram">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/images/histogram.png</normaloff>:/images/images/histogram.png</iconset>
   </property>
   <property name="text">
    <string>Histogram</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
